doc/kubernetes/modules/ROOT/pages/running/network-partition.adoc

```diff
@@ -0,0 +1,112 @@
+= Out-of-sync datacenters Recovery
+:description: This guide describes the steps to bring the secondary site up to date to the primary site after a network partition.
+
+include::partial$running/infinispan-attributes.adoc[]
+
+{description}
+
+// Infinispan -> clear caches on secondary site and perform the state transfer
+// Keycloak -> clear caches (how?)
+// Aurora -> ?
+// Route53 -> no-op
+
+== Audience
+
+This guide describes the procedures required to synchronize the secondary site after a temporary disconnection between sites.
+
+See xref:running/index.adoc[] for additional guides.
+
+=== {ispn} Cluster
+
+For the context of this guide, `{site-a}` is the primary site and `{site-b}` is the secondary site.
+
+Network partitions may happen between the site and the replication between the {ispn} cluster will stop.
+Manual steps described in this guide bring both site back in sync.
+
+include::partial$running/infinispan-batchcr-intro.adoc[]
+
+The `ConfigMap` with the {ispn} CLI operations are as follows:
```

Comment:
FYI Batch CRs can be created without a ConfigMap, e.g.:
```yaml
apiVersion: infinispan.org/v2alpha1
kind: Batch
metadata:
  name: mybatch
spec:
  cluster: infinispan
  config: |
    create cache --template=org.infinispan.DIST_SYNC mycache
    put --cache=mycache hello world
    put --cache=mycache hola mundo
```
IMO this is simpler for the user, but maybe there was a reason you always created a dedicated ConfigMap?

Reply:
> I had a call with Alexander and we thought about dropping the Batch CR, since we need the output of those commands, and use the CLI tool directly (`bin/cli.sh` from the pod or the `Route` endpoint).

+1, that would be simpler.